### DISCUSSION TITLE
How to use an background image?

### DISCUSSION THREAD

**Discussion by DennisGlobal** [ipxe/ipxe#802] [Q&A]
Hi all,

I wanted to use a background image for the iPXE boot menu.
So i added the following lines to our "boot.ipxe"

which is located in /tftpboot/menu/

console --picture http://10.10.10.1/background.png
colour --basic 3 --rgb 0xffaa44 3
cpair --foreground 0 --background 3 2
cpair --foreground 3 3

All textual edits are visible on the PXE boot,
but the background stays blue.

The Forum is locked, and i couldnt find a similair issue on the web.
So thats why i posted the question here now.

**Comment by oschroeder8** [comment]
Are you using a pre-built `boot.ipxe` file or are you compiling it yourself? Did you enable the `console_framebuffer` option in your local config file?

Check the following for details.
- https://ipxe.org/cmd/console
- https://ipxe.org/buildcfg/console_framebuffer

**Reply by DennisGlobal** [reply]
Yes, i use a pre-build one.
The version is even still on 1.0.0 (so, ill have to get around on updateing it)
I only edit the options in the boot.ipxe with atom.

**Comment by NiKiZe** [comment]
What happens when you run console --picture command? Any chance you are ignoring any error that shows?

Since you have enabled console option, you should have seen this full information? https://ipxe.org/cmd/console#build_options

As it says in the forum both in FAQ and other places, it is closed. And discussions are redirected here.

**Reply by DennisGlobal** [reply]
as far as i see/know there arent any errors.
i'll boot the machine later today, and poke around it

its to bad that the forum is closed,
iPXE works great, but the how-to part of it is hard to find.

but i also have the idea that im using it very basic, So ill have my work cut-out for me i see

**Reply by NiKiZe** [reply]
What is bad about the forum not being used when we have GitHub discussions?

**Comment by DennisGlobal** [comment]
Just an update:

I cán use an background image as mentioned here .
bút there are limitations:

mainly:

1024x768	OK (this is also the size of the 'old' image that was there)
1279x720	FAULT - no devices found
1024x576	OK (scaling with ratio)

as i noticed, the 1024 is leading on accepting the image
